Package.swift

```diff
@@ -62,7 +62,8 @@ let package = Package(
             targets: ["Realm", "RealmSwift"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/realm/realm-core", .exact(Version(coreVersionStr)!))
+        //.package(url: "https://github.com/realm/realm-core", .branch("lm/os-set"))
+        .package(path: "../realm-core")
```

Comment:
is it belongs to the local changes only?

Reply:
Yep, I need a PR merged in core